<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestionpatient</class>
 <widget class="QDialog" name="gestionpatient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>723</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion Patient</string>
  </property>
  <widget class="QLabel" name="male">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>500</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="stela">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>540</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>920</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
   </property>
   <item>
    <property name="text">
     <string>ID_patient</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Prenom</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Nom</string>
    </property>
   </item>
  </widget>
  <widget class="QTableView" name="tabpatient">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>160</y>
     <width>531</width>
     <height>331</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="excel">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>EXCEL</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>541</width>
     <height>591</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>11</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_9">
    <attribute name="title">
     <string>Supprimer</string>
    </attribute>
    <widget class="QPushButton" name="suppbuton_2">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>230</y>
       <width>181</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(255, 0, 0);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Supprimer</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>110</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Selectioner le ID a supprimer</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_10">
    <attribute name="title">
     <string>Ajouter</string>
    </attribute>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Id_patient  :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_29">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>180</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Prenom : </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_30">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>250</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_31">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>320</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Num_tel :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_32">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>390</y>
       <width>71</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Email :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_17">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>40</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>             ID &lt; 999</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_18">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>110</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_19">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>180</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_20">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>250</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_21">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>320</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="placeholderText">
      <string>  +216</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_22">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>390</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="insert_2">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>480</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Ajouter</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_11">
    <attribute name="title">
     <string>Modifier</string>
    </attribute>
    <widget class="QLabel" name="label_33">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Id_patient :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_34">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>130</y>
       <width>71</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">OPushButon{
 			border-radius: 15px;
			border-right:. 1px solid #aaaaaa;
			border-bottom: 1px solid #aaaaaa;
}

OPushButton:enabled {
background-color: blue;
color: white;
}
OPushBution-pressed {
background-color: #0d2f72;
color: #ffffe:
}
QPushButton:hover:!pressed{
background-Color: #E1F4FF;
Color: #0c2f70;
}
OPushButton:disabled {
background-color: #aaaaaa:
Color; #ffffff
}</string>
     </property>
     <property name="text">
      <string>Nom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_35">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Prenom :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_36">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>260</y>
       <width>91</width>
       <height>51</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Adresse :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_37">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>330</y>
       <width>91</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Num tel :</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_38">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>400</y>
       <width>81</width>
       <height>41</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>11</pointsize>
       <weight>50</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>Email :</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mod1_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>51</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mod2_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>120</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mod3_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>190</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QLineEdit" name="mod4_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>260</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mod5_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>330</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="mod6_3">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>400</y>
       <width>211</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="modifer_2">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>480</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>10</pointsize>
       <weight>10</weight>
       <italic>false</italic>
       <bold>false</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
     </property>
     <property name="text">
      <string>Modifier</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="idsup_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>530</y>
       <width>20</width>
       <height>20</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">
font: 75 6pt &quot;MS Shell Dlg 2&quot;;





background-color:rgb(255, 255, 255) ;
font 700 10pt &quot;segoe Print&quot; ;
color:white ;
border-radius : 10px ;</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_12">
    <attribute name="title">
     <string>statistique </string>
    </attribute>
    <widget class="QWidget" name="STAT_3" native="true">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>100</y>
       <width>561</width>
       <height>331</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="male_2">
   <property name="geometry">
    <rect>
     <x>840</x>
     <y>100</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="text">
    <string>Tri par</string>
   </property>
  </widget>
  <widget class="QPushButton" name="malek12">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>540</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial</family>
     <pointsize>10</pointsize>
     <weight>10</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(0, 85, 255);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>verifier captcha</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pdf">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>110</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 16pt &quot;Modern&quot;;
                 
background-color: rgb(255, 0, 0);
                       border: 2px solid black;
                       color: white;
                       padding: 5px;
border-radius:10px;
font: 87 10pt &quot;Arial&quot;;</string>
   </property>
   <property name="text">
    <string>PDF</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit45">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>40</y>
     <width>361</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);
                       border: 2px solid black;
                       color: black;
                       padding: 5px;
                       border-radius:10px;
font: 75 8pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="placeholderText">
    <string>Recherche</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
